src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -491,31 +611,32 @@ public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
 
       if (archive) {
         new SnackbarAsyncTask<Long>(getView(),
-                                    getResources().getQuantityString(R.plurals.ConversationListFragment_moved_conversations_to_inbox, 1, 1),
-                                    getString(R.string.ConversationListFragment_undo),
-                                    getResources().getColor(R.color.amber_500),
-                                    Snackbar.LENGTH_LONG, false)
+                getResources().getString(R.string.ConversationListFragment_deleting),
+                "",
```

Comment:
I'm guessing the second parameter is the default if the load doesn't work? Could we default to the english translation instead of an empty string?

Reply:
The second param is if you want a text that when clicked call the reverse action.  I thought about implementing the reverse action but it would take a lot of effort, because things are remove from many separate data stores. So id either have to delay the delete, and have the button cancel it but this would look bad ui wise, or save everything and reload if the button is clicked which is hard and possible not possible.

Reply:
I was thinking you could use the local database to cache the conversation instead of using their delete method, but we can consider this as an addon to this feature later.